frontend/src/layout.ejs

```diff
@@ -7,8 +7,13 @@
         {% include "head.html" %}
 
         <script>
+            const posthogJSApiKey = "{{ js_posthog_api_key|escapejs }}";
+            let posthogHost = {{ js_posthog_host|default:"null" }};
+            if (posthogHost === null) {
+                posthogHost = window.location.origin;
+            }
```

Comment:
`escapejs` https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#escapejs is the next thing to use instead of `|safe`